tensorflow/tools/ci_build/builds/check_system_libs.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Checks that the options mentioned in syslibs_configure.bzl are consistent with the valid options in workspace.bzl
+# Expects the tensorflow source folder as the first argument
+
+import sys
+import os
+from glob import glob
+
+tf_source_path = sys.argv[1]
+
+if not os.path.isdir(tf_source_path):
+  raise ValueError('The path to the TensorFlow source must be passed as'
+                   ' the first argument')
+
+syslibs_configure_path = os.path.join(tf_source_path, 'third_party',
+                                      'systemlibs', 'syslibs_configure.bzl')
+workspace_path = os.path.join(tf_source_path, 'tensorflow', 'workspace.bzl')
+third_party_path = os.path.join(tf_source_path, 'third_party')
+third_party_glob = os.path.join(third_party_path, '*', 'workspace.bzl')
+
+# Stub only
+def repository_rule(**kwargs):
+  del kwargs
+
+# Populates VALID_LIBS
+with open(syslibs_configure_path, 'r') as f:
```

Comment:
Can this section move into a function instead and return the set out? Currently it'll pollute global scope which isn't as safe.